.github/workflows/main.yml

```diff
@@ -0,0 +1,273 @@
+          # The default PostgreSQL port
+          POSTGRES_PORT: 5432
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Lint
+        run: |
+          sudo apt-get update && sudo apt-get install make
+          make lint ACTIONS=true
+
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Documentation
+        run: |
+          sudo apt-get update && sudo apt-get install make
+          make docs
+  
+  deps:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Cache pip dependencies
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-pip-deps-warehouse-p
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-${{ env.CACHE_SEED }}-${{ github.job }}-${{ env.cache-name }}-${{ hashFiles('requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ env.CACHE_SEED }}-${{ github.job }}-${{ env.cache-name }}-
+            ${{ runner.os }}-${{ env.CACHE_SEED }}-${{ github.job }}-
+            ${{ runner.os }}-${{ env.CACHE_SEED }}-
+            ${{ runner.os }}-
+
+      - name: Set cache permissions
+        run: |
+          mkdir ~/.cache && mkdir ~/.cache/pip
+          chown -R $(whoami) ~/.cache/pip
+
+      - name: Check dependencies
+        run: |
+          sudo apt -y install libcurl4-openssl-dev libssl-dev pkg-config
+          make deps
+
+  licenses:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Check licenses
+        run: bin/licenses
+  
+  translations:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Check translations
+        run: make translations
+
+  static-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Static Lint Check
+        run: |
+          npm install eslint --save-dev
+          ./node_modules/.bin/eslint 'warehouse/static/js/**' '**.js' 'tests/frontend/**' --ignore-pattern 'warehouse/static/js/vendor/**'
+          ./node_modules/.bin/sass-lint --verbose
+
+  static-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules-static-dh
```

Comment:
Same question here and below about the `-dh`

Reply:
Same as [the comment above](https://github.com/pypa/warehouse/pull/8255#discussion_r468752304). TLDR: shouldn't be there.